cap_anndata_schema.md

```diff
@@ -0,0 +1,1577 @@
+  		<td><b>example</b></td>
+  		<td><code>'EFO:0009922'</code> or <code>'EFO:0008931'</code></td>
+	</tr>
+</tbody></table>
+
+
+### assay
+
+<table><tbody>
+	<tr>
+  		<td><b>column</b></td>
+  		<td><code>assay</code></td>
+	</tr>
+	<tr>
+  		<td><b>dtype</b></td>
+  		<td>string</td>
+	</tr>
+	<tr>
+  		<td><b>value</b></td>
+  		<td>This MUST be the human-readable term which corresponds to the value of <code>disease_ontology_term_id</code>. The ontology term and ontology term ID MUST match.</td>
+	</tr>
+	<tr>
+  		<td><b>source</b></td>
+  		<td>file</td>
+	</tr>
+	<tr>
+  		<td><b>required for publication on CAP</b></td>
+  		<td>yes</td>
+	</tr>
+	<tr>
+  		<td><b>example</b></td>
+  		<td><code>'10x 3' v3'</code> or <code>'Smart-seq2'</code></td>
+	</tr>
+</tbody></table>
+
+
+### tissue_ontology_term_id
+
+<table><tbody>
+	<tr>
+  		<td><b>column</b></td>
+  		<td><code>tissue_ontology_term_id</code></td>
+	</tr>
+	<tr>
+  		<td><b>dtype</b></td>
+  		<td>string</td>
+	</tr>
+	<tr>
+  		<td><b>value</b></td>
+  		<td>This MUST be the most accurate child of <a href="https://www.ebi.ac.uk/ols4/ontologies/uberon/classes?obo_id=UBERON%3A0001062"><code>UBERON:0001062</code></a> for <i>anatomical entity</i>.</td>
+	</tr>
+	<tr>
+  		<td><b>source</b></td>
+  		<td>file or UI</td>
+	</tr>
+	<tr>
+  		<td><b>required for publication on CAP</b></td>
+  		<td>yes</td>
+	</tr>
+	<tr>
+  		<td><b>example</b></td>
+  		<td><code>'UBERON:0000451'</code> or <code>'UBERON:0000966'</code></td>
+	</tr>
+</tbody></table>
+
+### tissue
+
+<table><tbody>
+	<tr>
+  		<td><b>column</b></td>
+  		<td><code>tissue</code></td>
+	</tr>
+	<tr>
+  		<td><b>dtype</b></td>
+  		<td>string</td>
+	</tr>
+	<tr>
+  		<td><b>value</b></td>
+  		<td>This MUST be the human-readable term assigned to the value of <code>tissue_ontology_term_id</code>. The ontology term and ontology term ID MUST match.</td>
+	</tr>
+	<tr>
+  		<td><b>source</b></td>
+  		<td>file</td>
+	</tr>
+	<tr>
+  		<td><b>required for publication on CAP</b></td>
+  		<td>yes</td>
+	</tr>
+	<tr>
+  		<td><b>example</b></td>
+  		<td><code>'prefrontal cortex'</code> or <code>'retina'</code></td>
+	</tr>
+</tbody></table>
+
+
+### Clustering
+
+Users may OPTIONALLY include a single field for clustering within AnnData files, or multiple fields denoting clustering, e.g. different clustering algorithms, multiple resolutions of clustering, etc.
+
+We therefore REQUIRE that clustering is clearly denoted within the AnnData file if it contains clustering fields.
+
+ScanPy has set an AnnData community standard of defining the `*.obs` value by the type of algorithm. e.g. the function `scanpy.tl.louvain` (documented [here](https://scanpy.readthedocs.io/en/stable/generated/scanpy.tl.louvain.html)) by default saves the clustering as `anndata.obs['louvain']`. Similarly, `leiden` (documented [here](https://scanpy.readthedocs.io/en/stable/generated/scanpy.tl.leiden.html)) is often encoded as `anndata.obs['leiden']`.
```

Comment:
Please also note that we support two more key names: `leiden` and `louvian`. It means that user could use these names without `cluster_` prefix.

Reply:
I go back and forth on this issue. 

If `leiden` and `louvain` exist, they should be the only clustering in the file. 
Use case: the user then adds another level of clustering to a file with only `leiden`. Won't we then ask the user to format the clustering appropriately?

This makes me think that we should force users to format the clustering in a certain way.

Thoughts? @mfutey @unbelivr @rm1113 ?

Reply:
@rm1113 @evanbiederstedt I think we should always have the `'cluster'` prefix to keep the naming structure the same across datasets. 

Also the explanation of the column naming is already quite long, do we want to add another section stating you don't have to follow the three part naming convention of <code>'cluster + _ + [ALGORITHM_TYPE] + _ + [SUFFIX]'</code> ?